hoomd/hpmc/integrate.py

```diff
@@ -78,16 +78,18 @@ class HPMCIntegrator(BaseIntegrator):
 
     Attributes:
         a (`TypeParameter` [``particle type``, `float`]):
-            Maximum size of rotation trial moves.
+            Maximum size of rotation trial moves
+            :math:`[\\mathrm{arbitrary \\ unit}]`.
```

Comment:
I think explicitly writing arbitrary unit here is confusing. @joaander I am not that familiar with hpmc moves, is there something more specific we can put here? I would be fine with dimensionless, but I think writing arbitrary unit could be confusing to users.

Reply:
The comment in the implementation for 3D is:
```
        // Frenkel and Smit reference Allen and Tildesley, referencing Vesley(1982), referencing
        // Marsaglia(1972). Generate a random unit quaternion. Scale it to a small rotation and
        // apply.
```

It is not obvious to me what units the value is in from the implementation - it is used as a scale factor that multiples a unit quaternion. I don't have the time right now to trace down the original references for the algorithm or determine the proper units. Feel free to read those references and suggest the proper value.

Reply:
In that case I think we can just leave it as dimensionless.

Reply:
OK, I made that change. Good to go?

Reply:
Thank you!!